### PR TITLE
Add BitPersona demo page (#1814)

### LINKED ISSUE #1814: Add BitPersona demo page
There is a demo page for this component in the playground project but the page URL did not add to the menu, and also it does not have a proper design.

### PR DESCRIPTION
**Pull request by saeedhalab**
This closes #1814

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿.title{
+    font-weight:600;
+    margin:20px 0 ;
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.scss:3]: plz use `rem`

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/compilerconfig.json [hunk 2]
```diff
@@ -1,134 +1,138 @@
...
+    "inputFile": "Pages/Components/DropDown/BitDropDownDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/RadioButtonGroup/BitRadioButtonGroupDemo.razor.css",
+    "inputFile": "Pages/Components/RadioButtonGroup/BitRadioButtonGroupDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Persona/BitPersonaDemo.razor.css",
+    "inputFile": "Pages/Components/Persona/BitPersonaDemo.razor.scss"
+  }
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/compilerconfig.json:137]: so you are another `code formatting` destroyer 😁
plz use the correct indenting (4 spaces not 2 spaces)

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@maryamhdr 
@Cyrus-Sushiant 
please review this PR

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+            Type = "bool",
+            DefaultValue = "false",
+            Description = "If true, show the special coin for unknown persona. It has '?' in place of initials, with static font and background colors.",
+        },
+        new ComponentParameter()
+        {
+            Name = "Size",
+            Type = "string?",
+            DefaultValue = "",
+            LinkType=LinkType.Link,
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:119]: Plz, add space before and after `=`

**Review by msynk — DISMISSED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+                        Value = "Violet=16",
+                    },
+                    new()
+                    {
+                        Name = "WarmGray",
+                        Description = "",
+                        Value = "WarmGray=20",
+                    },
+                },
+
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:350]: Plz, Remove blank line

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+<BitPersona ImageUrl=""https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-female.png""
+            Presence=@BitPersonaPresenceStatus.DND
+            Size=@BitPersonaSize.Size120
+            HidePersonaDetails=""!IsHideDetails""
+            Text=""Annie Lindqvist""
+            SecondaryText=""Software Engineer""
+            TertiaryText=""In a meeting""
+            OptionalText=""Available at 4:00pm""></BitPersona>
+
+<style>
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:492]: I think, we don't need style in the example.

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+            Name = "AllowPhoneInitials",
+            Type = "bool",
+            DefaultValue = "false",
+            Description = "Whether initials are calculated for phone numbers and number sequences.",
+        },
+         new ComponentParameter()
+        {
+            Name = "CoinSize",
+            Type = "int",
+            DefaultValue = " -1",
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:23]: Remove the redundant space

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+            Type = "string?",
+            DefaultValue = "",
+            Description = "Optional text to display, usually a custom message set. The optional text will only be shown when using size100.",
+        },
+        new ComponentParameter()
+        {
+            Name = "Presence",
+            Type = "BitPersonaPresenceStatus",
+            LinkType = LinkType.Link,
+            Href="#precence-status",
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:82]: add a space before and after the `=`

**Review by maryamhdr — DISMISSED** [review]

**Comment by msynk** [comment]
@maryamhdr & @Cyrus-Sushiant plz re-review 😁

**Comment by msynk** [comment]
@saeedhalab plz resolve the comments that you believe are resolved after pushing new changes.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+        {
+            Id = "precence-status",
+            Title = "BitPersonaPresence enum",
+            Description = "",
+            EnumList = new List<EnumItem>()
+            {
+                new()
+                {
+                    Name = "Away",
+                    Description = "",
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Persona/BitPersonaDemo.razor.cs:151]: Plz, Remove `Description` when set empty

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@Cyrus-Sushiant plz re-review

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**